conteudo/exercicios/7-exercicio-projeto-casa/index.html

```diff
@@ -34,9 +34,9 @@ <h3>Alunas</h3>
         </a>
         <p>Ada Ferreira Porto Cruz</p>
       </div>
-      <div class="container__aluna">
-          <a href="#" target="_blank">
-            <img class="container__aluna-img" src="https://via.placeholder.com/500" alt="">
+      <div class="Ana Carolina Vieira">
```

Comment:
Conforme a Vanessa observou, você acabou alterando a class também @AnaCarolVieira. A class não muda.
Espero ter ajudado! 👍

Reply:
É para manter dessa forma
```suggestion
      <div class="container__aluna">
```